plottr/apps/inspectr.py

```diff
@@ -18,6 +18,9 @@
 import argparse
 import logging
 from typing import Optional, Sequence, List, Dict, Iterable, Union, cast, Tuple
+
+from PyQt5.QtCore import QPoint
+from PyQt5.QtWidgets import QMenu
```

Comment:
This will break support for pyside(2) 

Please do all qt imports using qtpy via the `plottr/__init__.py` module

Reply:
thanks! done